src/api/cloudDb/cloudDb.v6.service.js

```diff
@@ -0,0 +1,16 @@
+angular.module('ovh-api-services').service('OvhApiCloudDBV6', ($resource, OvhApiCloudDBV6) => {
+  const interceptor = {
+    response(response) {
+      OvhApiCloudDBV6.resetAllCache();
+      return response.data;
+    },
+  };
+
+  return $resource('/cloudDB', {}, {
+    query: {
+      method: 'GET',
+      isArray: true,
+      interceptor,
```

Comment:
I don't think you need an interceptor for a `GET`

Reply:
The interceptor has been removed.Thanks :)